src/Microsoft.AspNet.Razor/Parser/CSharpCodeParser.Directives.cs

```diff
@@ -27,14 +28,27 @@ private void SetupDirectives()
             MapDirectives(LayoutDirective, SyntaxConstants.CSharp.LayoutKeyword);
         }
 
+        protected virtual void TagHelperPrefixDirective()
+        {
+            TagHelperDirective(
```

Comment:
why reuse this method for an additional, fairly-different purpose?  a new method for the prefix directive would be simpler and you would only need to change the existing method's name